deploy

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+set -euo pipefail
+
+VERSION=$1
+
+xcrun agvtool what-marketing-version
+xcrun agvtool new-marketing-version $VERSION
+
+carthage update
+carthage build --no-skip-current --platform iOS
+
+git tag $VERSION
+git push origin $VERSION
+
+pod repo push tradingticket *.podspec --verbose --allow-warnings
```

Comment:
I don't want to test this until we want to push a new version.
![woo](http://www.craveonline.com/assets/uploads/2016/11/coight.gif)